client/src/app/app/core/repositories/motions/motion-format.service.spec.ts

```diff
@@ -1,16 +0,0 @@
-import { TestBed } from '@angular/core/testing';
```

Comment:
Because of the double `/app/app` I'm guessing this is a rebase relict and deleted it - correct me if I'm wrong